dispatch/static/manager/src/js/pages/DashboardPage.js

```diff
@@ -1,14 +1,85 @@
 import React from 'react';
 import DocumentTitle from 'react-document-title';
+import { Link } from 'react-router'
+import { connect } from 'react-redux'
+import * as dashboardActions from '../actions/DashboardActions'
+import moment from 'moment'
 
-export default class DashboardPage extends React.Component {
+class DashboardPageComponent extends React.Component {
+
+  componentWillMount() {
+    this.props.fetchActions(this.props.token)
+    this.props.fetchRecent(this.props.token)
+  }
 
   render() {
+
+    //returns the user history if the data is loaded, else returns an empty list element
+    let historyList = this.props.actions.isLoaded
+    ? this.props.actions.data.map((elem,ind) => (
+      <li key={ind}>
+        <i className={`fa fa-${elem.icon}`} aria-hidden="true"></i>
+        {elem.meta.author}
+        {elem.meta.count == 1 ? ` ${elem.meta.action} ` : ` made ${elem.meta.count} ${elem.meta.action} to ` }
+        <a href={elem.meta.article_url}>{elem.meta.headline}</a>
```

Comment:
Can we change this to link to the article edit page in Dispatch instead of the live version?

You should be able to do something like this:

```js
<LinkButton to={`articles/${elem.meta.article_id}`}>{elem.meta.headline}</LinkButton>
```

You'll just have to add the article ID to the API response first :metal: